components/property_access.rst

```diff
@@ -410,16 +410,17 @@ Sometimes, adder and remover methods don't use the standard ``add`` or ``remove`
     {
         // ...
 
-        public function joinPeople(string $people): void
+        public function joinPeople(string $person): void
```

Comment:
Maybe that's because I am not a native speaker but joining a people sounds odd to me. What about using the word "team" instead?

Reply:
Updated.